micro_ros_diagnostic_updater/example/example_processor_updater.c

```diff
@@ -30,37 +30,39 @@ static const int16_t PROCESSOR_TEMPERATURE_TASK_ID = 0;
 static const int16_t PROCESSOR_LOAD_TASK_ID = 1;
 
 rcl_ret_t
-my_diagnostic_temperature(diagnostic_value_t * temp_kv)
+my_diagnostic_temperature(diagnostic_value_t * values, uint8_t * number_of_key_values)
 {
+  *number_of_key_values = 1;
```

Comment:
```suggestion
my_diagnostic_temperature(diagnostic_value_t * values, uint8_t * number_of_values)
{
  *number_of_values = 1;
```
... for consistency with the field name in the message type.

Reply:
Yes, seems I missed the examples while looking at it